### DISCUSSION TITLE
✅  Connection error with my SSH key

### DISCUSSION THREAD

**Discussion by mememe-nome** [isontheline/pro.webssh.net#447] [Q&A] [answered]
I use WebSSH app on my iPad.

I wanna add new connection using my SSH key, but there is an connection error (Unknown Private Key Format).
I've refered #425 discussion, and checked what format my SSH key is. 
Then, because the SSH key begin with "-----BEGIN RSA PRIVATE KEY-----", I suppose its format is pem.

How do I deal with this error?

Thanks.

**Comment by isontheline** [comment, marked as answer]
Hello @C2-now 👋

1. Thanks for referring to #425 🙏
2. Does your key have a passphrase? Then you need to check the box on the private key.
3. Could you also check WebSSH 16.1? 
https://testflight.apple.com/join/QSrBK59z

Have a nice day ☀️

**Reply by isontheline** [reply]
Hello @C2-now 

Any update on your trouble?

**Reply by mememe-nome** [reply]
Thank you for your support, @isontheline!
And, sorry for my late reply.

You are amazing, for your advice helped me solving without my referring to pass phrase. 

Thank you a lot!!

**Reply by isontheline** [reply]
@C2-now you're Welcome 😉

Thank you a lot for your feedback 🙏
